package.json

```diff
@@ -44,33 +44,37 @@
 		"prop-types": "^15.5.10"
 	},
 	"devDependencies": {
+		"@babel/core": "^7.0.0-beta.44",
+		"@babel/plugin-transform-react-jsx": "^7.0.0-beta.44",
+		"@babel/register": "^7.0.0-beta.44",
 		"ansi-styles": "^3.1.0",
-		"ava": "^0.21.0",
-		"babel-plugin-transform-react-jsx": "^6.24.1",
-		"babel-register": "^6.24.1",
 		"eslint-config-xo-react": "^0.13.0",
 		"eslint-plugin-react": "^7.1.0",
 		"sinon": "^2.3.4",
 		"strip-ansi": "^4.0.0",
 		"svg-term-cli": "^2.1.1",
-		"xo": "^0.18.2"
+		"xo": "^0.18.2",
+		"ava": "^1.0.0-beta.3"
```

Comment:
`ava` should come after `ansi-styles`, could you reinstall it so that npm puts it in the right place?